### PR TITLE
Add Broker id and remove age restriction

### PR DESCRIPTION
**Pull request by toledoal**
Adding the broker id and removing the age restriction

### CODE REVIEW

File: src/utils/dateParser.js [hunk 1]
```diff
@@ -1,18 +1,11 @@
...
   }
   const dob = new Date(Date.parse((dateOfBirthText || "").trim()));
   const today = new Date();
-  const year = today.getFullYear();
-  const month = today.getMonth();
-  const date = today.getDate();
-  const minDateOfBirth = new Date(year - minAge, month, date);
-  const maxDateOfBirth = new Date(year - maxAge, month, date);
-  if (dob > minDateOfBirth || dob < maxDateOfBirth) {
+  if (dob > today) {
```
Comment (shubhagupta10) [inline src/utils/dateParser.js:8]: Will you add minDate 13 condition in this same PR?

Comment (toledoal) [reply src/utils/dateParser.js:8]: correct, I'll update this here...

File: src/index.test.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ describe('SDK Modal can', () => {
   })
 
   it('init a smartprice button with params', () => {
-    const endpoint = `https://test.smartprice.myrx.io/sdk-modal/?fn=Phil&ln=Jones&em=phil.jones@email.com&ph=1234567890&dob=2000-08-21`;
+    const endpoint = `https://smartprice.myrx.io/sdk-modal/?fn=Phil&ln=Jones&em=phil.jones@email.com&ph=1234567890&dob=2000-08-21`;
```
Comment (shubhagupta10) [inline src/index.test.js:42]: keep the link to test url in test files

File: src/utils/dateParser.js [hunk 3]
```diff
@@ -11,10 +10,11 @@ function dateOfBirth(dateOfBirthText) {
   const month = today.getMonth();
   const date = today.getDate();
   const minDateOfBirth = new Date(year - minAge, month, date);
-  const maxDateOfBirth = new Date(year - maxAge, month, date);
-  if (dob > minDateOfBirth || dob < maxDateOfBirth) {
-    return null;
+
+  if (dob > minDateOfBirth) {
+    return { value: dob, error: 'You must be older than 13' };
```
Comment (shubhagupta10) [inline src/utils/dateParser.js:15]: Where do we show this error?

**Review by shubhagupta10 — APPROVED** [review]

**Pull request merged**